### ISSUE TITLE
Wrong EllipsoidalChunk::representative_point?

### ISSUE THREAD

**Issue by bangerth** [geodynamics/aspect#1202]
I think this function is wrong:
```
    template <>
    Point<3>
    EllipsoidalChunk<3>::representative_point(const double depth) const
    {
      const int dim = 3;
      Assert(depth >= 0, ExcMessage("Given depth must be positive or zero."));
      Assert(depth <= maximal_depth(),
             ExcMessage("Given depth must be less than or equal to the maximal depth of this geometry."));

      // Choose a point on the center axis of the domain
      Point<dim> p =
        (manifold.push_forward(Point<3>(southLatitude * numbers::PI/180,
                                        eastLongitude * numbers::PI/180,
                                        -bottom_depth))
         + manifold.push_forward(Point<3>(northLatitude * numbers::PI/180,
                                          westLongitude * numbers::PI/180
                                          , 0))) / 2;
      p /= p.norm();
      p *= get_radius(p) - depth;
      return p;
    }
```
It *may* actually work, but I think what we really want to do is *first* average the points in spherical coordinates, and *then* do the `push_forward` operation. This would guarantee that we really have the center of the domain.

What do you say, @MFraters ?

**Comment by MFraters** [comment]
Yes, you are right. To get the correct centre of the domain, it should be the other way around.

**Comment by bangerth** [comment]
Fixed by #1211.
